website/content/docs/internals/telemetry/metrics/secrets.mdx

```diff
@@ -0,0 +1,91 @@
+---
+layout: docs
+page_title: "Telemetry reference: Secrets metrics"
+description: >-
+  Technical reference for secrets telemetry values.
+---
+
+# Secrets telemetry
```

Comment:
There seems to be a lot of overlap between this page and database.mdx, maybe they can be merged?  Database secrets engines are still secrets engines, after all.

Reply:
There's overlap because the specific DBs are relevant to both. But there are DB metrics not relevant to secrets and secrets metrics not relevant to DBs, so I'd argue it makes sense to have both pages.